app/views/publish/courses/confirmation.html.erb

```diff
@@ -31,10 +31,14 @@
         <% summary_list.row(html_attributes: { data: { qa: "course__level" } }) do |row| %>
           <% row.key { "Level" } %>
           <% row.value { course.level.humanize } %>
-          <% row.action(
-            href: new_publish_provider_recruitment_cycle_courses_level_path(course.provider.provider_code, course.recruitment_cycle.year, params.to_unsafe_h.merge(goto_confirmation: true)),
-            visually_hidden_text: "level"
-          ) %>
+          <% if course.is_further_education? %>
+            <% row.action %>
+          <% else %>
+            <% row.action(
+              href: new_publish_provider_recruitment_cycle_courses_level_path(course.provider.provider_code, course.recruitment_cycle.year, params.to_unsafe_h.merge(goto_confirmation: true)),
+              visually_hidden_text: "level"
+            ) %>
+          <% end %>
```

Comment:
I think its a bad idea to allow for change link at any level

Also can we have tests

Reply:
To confirm, you think we should remove the change link for all levels of course, FE, Primary, Secondary ?

Reply:
Yes we should remove the change link for at levels.

There is not enough work done to support switching from level to level as there will be values passed over that is not relevant for the level specified.

Reply:
Could we also remove the change link for the SEND on this page? It'll essentially allow changing the level too

Reply:
Just saw this, will do